core/src/main/java/de/danoeh/antennapod/core/util/ShareUtils.java

```diff
@@ -59,17 +51,14 @@ public static boolean hasLinkToShare(FeedItem item) {
         return FeedItemUtil.getLinkWithFallback(item) != null;
     }
 
-    public static void shareFeedItemLink(Context context, FeedItem item, boolean withPosition) {
-        String text = getItemShareText(item) + " " + FeedItemUtil.getLinkWithFallback(item);
-        if (withPosition) {
-            int pos = item.getMedia().getPosition();
-            text += " [" + Converter.getDurationStringLong(pos) + "]";
+    public static void shareFeedItemLinkWithDownloadLink(Context context, FeedItem item, boolean withPosition) {
+        String text = getItemShareText(item) + " ";
+        if (hasLinkToShare(item)) {
+            text +=  "\n\nWebsite: " + FeedItemUtil.getLinkWithFallback(item);
+        }
+        if (item.getMedia() != null && item.getMedia().getDownload_url() != null) {
+            text += "\n\nMedia file: " + item.getMedia().getDownload_url();
```

Comment:
Sorry to complain again. Could you please load this text from the strings file, so it can be translated?